src/invidious/yt_backend/youtube_api.cr

```diff
@@ -201,8 +208,8 @@ module YoutubeAPI
         "gl"            => client_config.region || "US", # Can't be empty!
         "clientName"    => client_config.name,
         "clientVersion" => client_config.version,
-      },
-    }
+      } of String => String | Int64 | Nil,
```

Comment:
This hash shouldn't have any `Nil` elements in it (we're checking if the value is `Nil` with `.nil?`)